documentation/GETTING_STARTED.md

```diff
@@ -0,0 +1,349 @@
+
+> **_NOTE_**: You must be on Python version 3.9 or above. The installation may fail unexpectedly on older versions of Python.
+
+```bash
+git clone https://github.com/allen-cell-animated/colorizer-data.git
+cd colorizer-data/documentation/getting_started_guide
+
+python -m pip install -r ./requirements.txt
+```
+
+## 3. Expected formats for raw data
+
+For this tutorial, we'll be working with sample data included in the [`getting_started_guide/raw_dataset`](./getting_started_guide/raw_dataset/) directory.
+
+This dataset is a simplified example of raw, pre-processed segmentation data. The data was generated using the [`generate_raw_data.py` script](./getting_started_guide/scripts/generate_data.py), which generates a **CSV file** with columns for object IDs, track IDs, times, centroids, features (volume/height), and paths to the segmentation images. The **segmentation images** are 2D images in the OME-TIFF format, encoding the locations of segmented objects.
+
+Your files may be in a different format or have 3D segmentation images, in which case it will need to be transformed. Generally, we recommend:
+
+1. Save your data as a CSV or other format that can be read into a pandas `DataFrame`.
+2. Make every segmented object its own row in the table.
+3. Save track ID, time, centroids, and other information as columns.
+4. Create columns for any additional features you want to visualize.
+
+### What does the example dataset look like?
+
+Here's a preview of the raw dataset, `data.csv`:
+
+| object_id | track_id | time | centroid_x | centroid_y | area | radius | location | segmentation_path |
+| ----------- | ---------- | ------ | ------------ | ------------ | -------- | -------- | ------------------- | --- |
+| 1 | 0 | 0 | 33 | 110 | 706.9 | 15 | middle | frame_0.tiff |
+| 2 | 1 | 0 | 67 | 100 | 804.2 | 16 | middle | frame_0.tiff |
+| 3 | 2 | 0 | 100 | 108 | 804.2 | 16 | middle | frame_0.tiff |
+| 4 | 3 | 0 | 133 | 88 | 706.9 | 15 | middle | frame_0.tiff |
+| 5 | 4 | 0 | 167 | 101 | 804.2 | 16 | middle | frame_0.tiff |
+| 6 | 0 | 1 | 33 | 121 | 530.9 | 13 | bottom | frame_1.tiff |
+| 7 | 1 | 1 | 67 | 113 | 804.2 | 16 | middle | frame_1.tiff |
+| ... | ... | ... | ... | ... | ... | ... | ... | ... |
+
+Each of the segmentation images is an OME-TIFF image containing the IDs of the segmented objects:
+
+_Frame 0 of the example dataset, as viewed in FIJI. Contrast has been increased for easier viewing._
+
+![Frame 0 of the example dataset. The background is black, labeled `value=0`. There are five circles of various diameters and positions, with IDs starting at 30 and increasing to 34.](./getting_started_guide/assets/sample-segmentation.png)
+
+> **_NOTE:_** Note that `value=0` is used to represent the background in the segmentation images. For simplicity, we recommend starting object IDs at `1` to avoid conflicts with the background value.
```

Comment:
```suggestion
> **_NOTE:_** A value of `0` is used to represent the background in the segmentation images. For simplicity, we recommend starting object IDs at `1` to avoid conflicts with the background value.
```